be1-go/channel/coin/verification.go

```diff
@@ -0,0 +1,55 @@
+package coin
+
+import (
+	"golang.org/x/xerrors"
+	"popstellar/message/messagedata"
+	"strconv"
+)
+
+func (c *Channel) verifyMessageTransactionPost(transactionPost messagedata.TransactionPost) error {
```

Comment:
Since you don't need the Channel for the verification, this verification function should be implemented on the messagedata itself. Take inspiration from https://github.com/dedis/popstellar/blob/805265d9e38a7f019caa0aa557a568d43a2cf636/be1-go/message/messagedata/consensus_propose.go#L31.